src/app/lib/config/toggles/__snapshots__/index.test.js.snap

```diff
@@ -27,7 +27,7 @@ Object {
   },
   "liveRadioSchedule": Object {
     "enabled": true,
-    "value": "(hausa|korean)",
+    "value": "(afaanoromoo|afrique|amharic|arabic|bengali|burmese|hausa|indonesia|korean|nepali|pashto|persian|somali|swahili|tigrinya|urdu)",
```

Comment:
not sure I understand the value of this snapshot but 🤷‍♂️

Reply:
Let's you know what you changed :P